features/nanostack/mbed_lib.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "nanostack-upper"
+}
```

Comment:
As explained here: https://github.com/ARMmbed/mbed-os/pull/9628#pullrequestreview-200606675

We don't need the "nanostack-upper" module, as it remains empty.

Reply:
Would you like me to take the commit from #9628 into this PR?

Reply:
Yes...maybe...